promql/engine.go

```diff
@@ -331,9 +352,8 @@ func (ng *Engine) exec(ctx context.Context, q *query) (Value, error) {
 	defer q.cancel()
 
 	const env = "query execution"
-
-	evalSpanTimer, ctx := q.stats.GetSpanTimer(ctx, stats.EvalTotalTime)
-	defer evalSpanTimer.Finish()
+	evalTimer := q.stats.GetTimer(stats.EvalTotalTime).Start()
```

Comment:
Why does the function change here?

Reply:
I think I picked the wrong change when fixing a merge conflict here during rebase, fixing.